android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/oasis/TestOasisSigner.kt

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+package com.trustwallet.core.app.blockchains.oasis
+
+import com.google.protobuf.ByteString
+import com.trustwallet.core.app.utils.Numeric
+import com.trustwallet.core.app.utils.toHexByteArray
+import com.trustwallet.core.app.utils.toHexBytes
+import com.trustwallet.core.app.utils.toHexBytesInByteString
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+import wallet.core.java.AnySigner
+import wallet.core.jni.CoinType.OASIS
+import wallet.core.jni.proto.Oasis
+import wallet.core.jni.proto.Oasis.SigningOutput
+import java.math.BigInteger
+
+class TestOasisSigner {
+
+    init {
+        System.loadLibrary("TrustWalletCore")
+    }
+
+    @Test
+    fun OasisTransactionSigning() {
+        val transferMsg = Oasis.TransferMessage.newBuilder()
+
+        transferMsg.apply {
+            to = "oasis1qrrnesqpgc6rfy2m50eew5d7klqfqk69avhv4ak5"
+            gasPrice = 0
+
+            // This is the way Oasis indicates sign of the amountd
+            // https://github.com/oasisprotocol/oasis-core/blob/483bd3a897454e4bc1a8795675e7f29cd4d8e72d/go/common/quantity/quantity.go#L39-L55
+            gasAmount = "0"
+            
+            nonce = 0
+            method = "staking.Transfer"
+
+            // This is the way Oasis indicates sign of the amount
+            // https://github.com/oasisprotocol/oasis-core/blob/483bd3a897454e4bc1a8795675e7f29cd4d8e72d/go/common/quantity/quantity.go#L39-L55
+            amount = "10000000" // decimal 10000000
```

Comment:
Let's move this comment to cpp code
```suggestion
            amount = "10000000"
```

Reply:
Done!